### PR TITLE
Pin notes

### PR DESCRIPTION
**Pull request by audreylemeur**

### CODE REVIEW

File: client/src/app/trash/trash.component.html [hunk 1]
```diff
@@ -4,7 +4,7 @@ <h2>Trash Can</h2>
 
 <div *ngIf="notes; else notesError">
 <ul class= "note-container">
-  <mat-card class="note-card" *ngFor="let note of this.notes">
+  <mat-card class="unpinned-note-card" *ngFor="let note of this.notes">
```
Comment (ErikRauer) [inline client/src/app/trash/trash.component.html:7]: We should unpin notes when we move them to the trash

File: client/src/app/notes.service.ts [hunk 2]
```diff
@@ -13,7 +13,8 @@ import { map } from 'rxjs/operators';
 export class NotesService {
 
   readonly noteUrl: string = environment.API_URL + 'notes';
-  readonly deleteNoteUrl: string = environment.API_URL + 'notes/delete'
+  readonly deleteNoteUrl: string = environment.API_URL + 'notes/delete';
+  readonly pinNoteUrl: string = environment.API_URL + "notes/pin";
```
Comment (ErikRauer) [inline client/src/app/notes.service.ts:17]: Probably shouldn't be double quotation marks for the sake of consistency

File: client/src/app/trash/trash.component.scss [hunk 3]
```diff
@@ -12,15 +12,15 @@
...
+.unpinned-note-card .mat-card-actions {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
 }
 
 // For some reason, Angular gives margins to every button but the first.
 // So, we have to go back and add these margins manually.
-.note-card .mat-card-actions .mat-button:first-child {
+.unpinned-note-card .mat-card-actions .mat-button:first-child {
```
Comment (ErikRauer) [inline client/src/app/trash/trash.component.scss:23]: Don't we have a `notes.scss` to do this in?

Comment (ErikRauer) [reply client/src/app/trash/trash.component.scss:23]: Might be something to take note of for refactoring in the future

Comment (floogulinc) [reply client/src/app/trash/trash.component.scss:23]: If its global to multiple components then it should just be in `styles.scss`. But really if you are reusing cards like that the card itself with all its styles should be its own component with the data to display in it passed in.

Comment (audreylemeur) [reply client/src/app/trash/trash.component.scss:23]: It's only used on owner page

**Pull request merged**